### PR TITLE
Frontend for Create Section page

### PR DESCRIPTION
**Pull request by fredy20501**

### CODE REVIEW

File: src/components/CreateSectionForm.vue [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+    <form @submit.prevent>
+      <!-- Put fields & buttons with their labels here -->
+        <div >
+          <ValidationProvider name="Section Tag" rules="required" v-slot="{ errors }">
+            <label for="sectionTags">Section Tag</label><br>
+            <input list= "sectionTags" name="sectionTag" type="text" style="width:300px;" v-model="sectionTag">
+            <datalist id=sectionTags>
+              <!--Example, should import user's sections from db-->
+              <option value=ADAM-ECE3221-FRO1B-LEC></option>
+            </datalist>
```
Comment (fredy20501) [inline src/components/CreateSectionForm.vue:15]: The "Section Tag" is just the name of the course. It isn't a dropdown and there aren't any choices so there is no need for the <datalist> tag.

File: src/components/CreateSectionForm.vue [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+            <span v-if="errors.length" class="error">{{ errors[0] }}</span>
+          </ValidationProvider>
+          <br><br>
+          <button 
+            style="width:300px;"
+            v-on:click="$.createSection()"
```
Comment (fredy20501) [inline src/components/CreateSectionForm.vue:21]: In the vue attributes you can jut put the name of the function directly.
```suggestion
            v-on:click="createSection"
```

**Comment by fredy20501** [comment]
Here is what the frontend looks like currently:
image

**Comment by fredy20501** [comment]
I still have to do the API calls to connect to the backend but I will set this as ready for review so people can start reviewing everything that is already there while I do the finishing touches.

File: src/store.js [hunk 3]
```diff
@@ -127,6 +127,66 @@ const store = new Vuex.Store({
...
+      // return $http.get('seatingPlan')
+      // .catch(err => {
+      //   err.message = "Could not get seating layouts. Please try again later"
+      //   throw err
+      // })
+
+      // Return mock data since api is not here yet
+      return new Promise((resolve) => {
+        resolve({
+          seatingLayouts: [
```
Comment (ravenmills3) [inline src/store.js:142]: Good idea to include mock data, in the event that things do not get completed for demo tomorrow

**Review by ravenmills3 — APPROVED** [review]
Really great work by the Frontend and UI team

**Comment by fredy20501** [comment]
Integration with the backend API is complete: everything is functional!

**Pull request merged**